tests/suite/math/delimited.typ

```diff
@@ -62,3 +62,7 @@ $ 1/(2 y (x) (2(3)) $
 // Test ignoring weak spacing immediately after the opening
 // and immediately before the closing.
 $ [#h(1em, weak: true)A(dif x, f(x) dif x)sum#h(1em, weak: true)] $
+
+--- math-lr-corner-brackets ---
+// Test positioning of U+231C to U+231F
```

Comment:
could you add a comment here that they are supposed to look the same? the "vs" had me briefly thinking that they are supposed to look _different._

could also rename it to `issue-4188-lr-corner-brackets` to link it to that issue

Reply:
I’ve changed the `vs` to an `=`.